wodles/vuls/vuls.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+################################################################################
+# Wazuh wrapper for Vuls
+# Wazuh Inc.
+# Dec 19, 2017
+################################################################################
+
+
+from datetime import datetime
+from socket import socket, AF_UNIX, SOCK_DGRAM
+from subprocess import call
+import json
+import sys
+import getopt
+
+# Enable/disable debug mode
+enable_debug = 0
+# Wazuh installation path
+wazuh_path = open('/etc/ossec-init.conf').readline().split('"')[1]
+# Vuls installation path
+vuls_path = '{0}/wodles/vuls'.format(wazuh_path)
+# Wazuh queue
+wazuh_queue = '{0}/queue/ossec/queue'.format(wazuh_path)
+# Path to VULS logs
+vuls_log = '{0}/logs/vuls/'.format(wazuh_path)
+# Path to VULS binary
+vuls_bin = '{0}/go/bin/vuls'.format(vuls_path)
+# Path to CVE fetcher
+cve_db_fetcher = '{0}/go/bin/go-cve-dictionary'.format(vuls_path)
+# Path to OVAL fetcher
+oval_db_fetcher = '{0}/go/bin/goval-dictionary'.format(vuls_path)
+# Path to VULS config
+vuls_config = '{0}/config.toml'.format(vuls_path)
+# Path to CVE database
+cve_db = '{0}/cve.sqlite3'.format(vuls_path)
+# Path to OVAL database
+oval_db = '{0}/oval.sqlite3'.format(vuls_path)
+
+def help():
+    print('wazuh-vuls \n' \
+    '           [--mincvss 5]               Minimum score to report.\n' \
+    '           [--updatenvd]               Update NVD database.\n' \
+    '           [--nvd-year 2002]                Year from which the CVE database will be downloaded'
+    '           [--updaterh]                Update Redhat OVAL database.\n' \
+    '           [--updateub]                Update Ubuntu OVAL database.\n' \
+    '           [--updatedeb]               Update Debian OVAL database.\n' \
+    '           [--updateorac]              Update Oracle OVAL database.\n' \
+    '           [--autoupdate]              Oval database auto-update.\n' \
+    '           [--os-version]              OS version for downloading the OVAL database'
+    '           [--onlyupdate]              Only update databases.\n' \
+    '           [--source <nvd|oval>]       CVE database preferred. The default will be the one that takes the highest CVSS.\n' \
+    '           [--antiquity-limit 30]      Warn if vulnerability update date is less than X days.\n' \
+    '           [--disable-package-info]    Disable packages info.\n' \
+    '           [--debug]                   Debug mode.\n')
+
+def extract_CVEinfo(cve, type):
+    if type == 'nvd':
+        source = 'National Vulnerability Database'
+    elif type == 'redhat' or type == 'centos':
+        source = 'RedHat OVAL'
```

Comment:
Add this line here:
```python
type = redhat
```

This will prevent this function from accessing `[ScannedCves][CveContents][centos]`